### ISSUE TITLE
[TBC The Slave Pens] Claw (Musel'ek pet) does not turn friendly at low health so the quest to find him can't be turned in

### ISSUE THREAD

**Issue by elthehablo** [azerothcore/azerothcore-wotlk#16712]
Originally reported on CC at: https://github.com/chromiecraft/chromiecraft/issues/5725
By: [Kirishiki](https://github.com/Kirishiki)

Likely happens because of the mismatch of Claw still being in SAI and the boss now being in script. Perhaps we can try putting Claw and at least what he does in combat in script too.

### What client do you play on?

enUS

### Faction

Both

### Content Phase:

70

### Current Behaviour

the boss fight with swamplord musel ek and claw, the bear should turn into friendly npc at low hp and be talked to for getting credit for quest "lost and action" but he just doesnt turn and dies.

### Expected Blizzlike Behaviour

the bear turns into friendly windcaller claw at low hp and u can talk to him for credit for quest "lost in action"

### Source

???

### Steps to reproduce the problem

1. go to underbog
2. go to boss swamplord musel ek and claw
3. kill boss and start nuking the bear
4. when hes low hp ull realise hes not gonna get any friendlier and keeps ferociously biting ya
5. u deside to let him out of his misery and dwell in agony for no quest credit

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/5c31998573613c7564d4d5a9b050d2656b0e4c23

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Gultask** [comment]
Claw remains in SAI? I can do something real quick, then.

**Comment by Gultask** [comment]
Only problem will be reset, although we'll deal with it later or in Muse'lek's script with SET_DATA or DO_ACTION

**Comment by elthehablo** [comment]
> Claw remains in SAI? I can do something real quick, then.

Yes I had some failed attempts to get him in script but ran out of time at the time

**Comment by Gultask** [comment]
Updated the issue because I can't be bothered to check this rn but it should be easy to add in any case 😩
btw, added to CC live already.

**Comment by v-mstrs** [comment]
Its in Underbog...

**Comment by elthehablo** [comment]
Worked well for me too.

**Comment by elthehablo** [comment]
Couldn't confirm the new issue tho
